dependencies.yaml

```diff
@@ -662,10 +658,12 @@ dependencies:
       - output_types: [conda, pyproject]
         packages:
           - pandas
+          - pydantic
```

Comment:
Curious what `pydantic` is being used for

Reply:
DGL requires it.  It tries to import it and raises an exception if it isn't available.  It's used by GraphBolt.